data/docs/livekit-observability.mdx

```diff
@@ -0,0 +1,591 @@
+---
+date: 2025-11-25
+id: livekit-observability-with-opentelemetry
+tags: [SigNoz Cloud, Self-Host]
+title: LiveKit Observability with SigNoz
+description: Observability and monitoring for LiveKit performance and usage with OpenTelemetry instrumentation to send traces, logs, and metrics to SigNoz
+doc_type: howto
+---
+
+## Overview
+
+This guide walks you through setting up observability and monitoring for LiveKit using [OpenTelemetry](https://opentelemetry.io/) and exporting logs, traces, and metrics to SigNoz. With this integration, you can observe the performance of various models, capture request/response details, and track system-level metrics in SigNoz, giving you real-time visibility into latency, error rates, and usage trends for your LiveKit applications.
+
+Instrumenting LiveKit in your AI applications with telemetry ensures full observability across your voice agent workflows, making it easier to debug issues, optimize performance, and understand user interactions. By leveraging SigNoz, you can analyze correlated traces, logs, and metrics in unified dashboards, configure alerts, and gain actionable insights to continuously improve reliability, responsiveness, and user experience.
+
+## Prerequisites
```

Comment:
Add this drop-in snippet before prerequisites - https://github.com/SigNoz/signoz.io/blob/main/CONTRIBUTING.md?plain=1#L252